website/docs/reference/glossary.md

```diff
@@ -146,6 +146,12 @@ See [Operation modes](../configuration/operation-modes.md) for more details.
 An open source relational database.
 FerretDB uses PostgreSQL as a database engine.
 
+#### Pushdown
+
+The method of optimizing a query by reducing the amount of data read and processed in FerretDB, by moving some logic
+to underlying backend.
+See [Query pushdown](../pushdown.md) for more details.
```

Comment:
Don't use "query pushdown". Do use "filter pushdown" and "sort pushdown"

Reply:
Query pushdown is documentation page title. It's the general term for all types of pushdown. I don't understand what's wrong with it here, and what's the alternative?

Reply:
The alternative is "pushdown". It is always unclear, unlike "query pushdown" when is it sometimes unclear and sometimes means the same as "filter pushdown"

Reply:
Let's focus on code changes for now, I will take a look on docs

Reply:
>  "query pushdown" when is it sometimes unclear and sometimes means the same as "filter pushdown"

I thought that was the issue we want to solve. This PR removes all "Query Pushdown" occurrences that mean "Filter Pushdown".

The standalone "Pushdown" seems a bit unclear to me, as it doesn't state what is "pushed down". That's Why Query Pushdown seems a better fit for general term, especially it's already used by some people (Googling `query pushdown` results with db related articles, and `pushdown` results with triceps exercise :dizzy_face:)

Reply:
> Let's focus on code changes for now, I will take a look on docs

Okay! I will quickly remove docs changes and re-request review